cocos/renderer/pipeline/forward/ForwardPipeline.h

```diff
@@ -62,6 +62,7 @@ class CC_DLL ForwardPipeline : public RenderPipeline {
     void setSkybox(uint);
     void setShadows(uint);
     void destroyShadowFrameBuffers();
+    uint8_t getCombineSignY() const;
```

Comment:
please add some comments for combineSignY

Reply:
combineSignY is commented on the js side